tgui/packages/tgui/interfaces/ScannerGate.jsx

```diff
@@ -165,6 +170,11 @@ const ScannerGateOff = (props) => {
           content="Nutrition"
           onClick={() => act('set_mode', { new_mode: 'Nutrition' })}
         />
+        <Button
+          content="Contraband"
+          disabled={contraband_enabled ? true : false}
```

Comment:
Wait, `disabled={contraband_enabled}`? Something's definitely backwards here, it might be the var name itself (referring to my earlier review about the dm side of this)